cmd/registry/pull/pull_test.go

```diff
@@ -297,7 +297,7 @@ var registryPullTests = Describe("pull", func() {
 				args = []string{registryCmd, pullCmd, newReg, "--plain-http", "--config", configFile}
 			})
 			pullAssertFailedBehavior(registryPullUsage, fmt.Sprintf("ERROR unable to create new repository with ref %s: "+
-				"invalid reference: invalid digest; invalid checksum digest format\n", newReg))
+				"invalid reference: invalid digest %q: invalid checksum digest format\n", newReg, "something"))
```

Comment:
is this correct? why not add the hard coded var straight in the string?

Reply:
Just a preference to let `Sprintf` quote the string for me.

Reply:
humm, ok